src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
```

Comment:
@ChristophTobler 
Is this what you suggested? 
And tty-1 doesn't work for your board? It works for my eagle board. Are we using different serial port? Thanks.

#if defined(__PX4_QURT)
// Snapdragon: use J15 (next to USB)
#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-1"
#else
// Snapdraogon: use J12 (next to J13, power module side)
#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-3"
#endif